parquet-hadoop/src/main/java/org/apache/parquet/hadoop/util/ContextUtil.java

```diff
@@ -251,17 +251,36 @@ public static Configuration getConfiguration(JobContext context) {
     }
   }
 
-  public static Counter getCounter(TaskInputOutputContext context,
-                                   String groupName, String counterName) {
-    return (Counter) invoke(GET_COUNTER_METHOD, context, groupName, counterName);
```

Comment:
I'll split the difference and cache the counter method the first time it's used. Thanks for the catch.